src/main/java/com/crud/library/domain/User.java

```diff
@@ -0,0 +1,32 @@
+package com.crud.library.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Entity(name = "USERS")
+public class User {
+
+    @Id
+    @GeneratedValue
+    private long id;
+    private String firstName;
+    private String lastName;
+    private String mailAddress;
+    private LocalDate signUp;
+    @OneToMany(
```

Comment:
potrzebne mapowanie dwukierunkowe?

Reply:
a nie jest tak, ze użytkownik może mieć wiele wypożyczeń konkretnych exemplarow?

Reply:
Moze, tylko czy potrzebujemy relacji dwukierunkowej? Poczytaj sobie one-directional oraz bi-directional mappings in Hibernate, bo nie rozumiesz o czym pisze ;)